views/loan-personal.hbs

```diff
@@ -454,9 +455,22 @@ POST /v2.1/accounts/{accountId}/envelopes/{envelopeId}/views/recipient
   "email": "-- RECIPIENT EMAIL HERE --",
   "recipientId": "1",
   "returnUrl": "-- RETURN URL HERE --",
-  "authenticationMethod": "email"
+  "authenticationMethod": "email",
+  "frameAncestors": ["-- DOMAIN NAME –-", “https://apps-d.docusign.com”],
+  "messageOrigins": ["https://apps-d.docusign.com"]
+
 }
 </pre>
+</div>
+<h5>
+          Step 3
+        </h5>
+
+        <p>
+          We next embed the envelope on the page. To do this, we place a <code>div</code> container on the html page, and then use the DocuSign JS library to mount the envelope to the page. We will need to provide the signing URL from the previous step and the div's id where the document will be rendered. We can also customize the <code>finishText</code>, <code>position</code>, and <code>signingNavigation</code> button for focused view signing.
+        </p>
+
+        <p>Learn how to set up your HTML page and create a focused view script using the DocuSign JS library on the <a href='https://developers.docusign.com/docs/esign-rest-api/esign101/concepts/embedding/#focused-view' target='_blank'>DocuSign Developer Center.</a></p>
```

Comment:
This link goes to the guide, do we have a link to the How-To elsewhere? if not - can we switch this to a link to the how-to?